lnwallet/channel.go

```diff
@@ -4273,7 +4276,8 @@ func (lc *LightningChannel) FailHTLC(htlcIndex uint64, reason []byte) error {
 // commitment update. This method is intended to be called in order to cancel
 // in _incoming_ HTLC.
 func (lc *LightningChannel) MalformedFailHTLC(htlcIndex uint64,
-	failCode lnwire.FailCode, shaOnionBlob [sha256.Size]byte) error {
+	failCode lnwire.FailCode, shaOnionBlob [sha256.Size]byte,
```

Comment:
The `godoc` comment should be updated to describe the semantics of the additional refs.